tests/utils/run-tests-gcp-vm.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
```

Comment:
I wonder if shell scripts and ssh'ing will be manageable as we increase the number of supported tests. 

I wonder if there are any tools we can use to:
- declaratively provision VMs and configure it with the needed config
- declaratively run a test on a VM or a set of VMs

Reply:
Sure, there are plenty of tools 😁 We could use an IaC tool (terraform, pulumi etc) but is that too much overhead for running locally? It means more dev dependencies.